app/services/gpo_confirmation_exporter.rb

```diff
@@ -54,7 +54,7 @@ def make_entry_row(confirmation)
   end
 
   def format_date(date)
-    "#{date.strftime('%-B %-e')}, #{date.year}"
+    date.utc.strftime('%-B %-e, %Y').to_s
```

Comment:
`Time#utc` mutates the caller which is a major bummer, also is the `.to_s` redundant?

```suggestion
    date.in_time_zone('UTC').strftime('%-B %-e, %Y')
```

Also file this away for the future, but looking at this I realized that it's not translated. But we don't want to translate just the dates like this because the entire letter template is hardcoded to English. So maybe for the future we'd want to track translating the letters.